### ISSUE TITLE
Epyx California Games on IIe Card: Surfing Screen Garbled

### ISSUE THREAD

**Issue by JDW1** [a2-4am/4sports#18]
When using an **Apple IIe Card** running in Color Mode at 1 MHz on a Macintosh LC575 motherboard with IIe Startup 2.2.2d1, Epyx California Games loads just fine.  
![IMG_3727](https://user-images.githubusercontent.com/3928027/202854912-fc89111a-9c0e-4e2e-8f9b-f879bdc65222.JPG)

I can then choose Surfing and play the game just fine.  
![IMG_3722](https://user-images.githubusercontent.com/3928027/202854945-0d656141-9e0b-49bf-bcb8-2b62772475f7.JPG)

But when I lose, it exists to show the following garbled screen...
![IMG_3716](https://user-images.githubusercontent.com/3928027/202854968-d4b22f56-7209-4884-9ba1-d492581290cc.JPG)

**Comment by txgx42** [comment]
I am getting the same garbled screen in AppleWin, so this one at least doesn't seem to be IIe Card specific.

![image](https://user-images.githubusercontent.com/19871324/202866720-5a1fb06d-3735-4ad2-a96f-957806b2e3be.png)

**Comment by xotmatrix** [comment]
Reproduced on real Apple //e.

4am crack and san inc prodos pack work OK.

**Comment by peterferrie** [comment]
I suspect that I've got a bad volume-change detection and it's reading half of the content from the wrong image.

**Comment by peterferrie** [comment]
4am crack on Asimov displays badly, too.  I will find a good copy.

**Comment by xotmatrix** [comment]
This is weird. I've played this quite a bit now and am still unable to reproduce the problem with the 4am crack. How are you managing it, qkumba?

![AppleWin_2022-12-22_00-15-57](https://user-images.githubusercontent.com/14209329/209062289-1e6b5db0-c467-44a7-b0a5-9f7fda3c4934.png)

**Comment by peterferrie** [comment]
It happens every time for me in both AppleWin and MAME.

**Comment by peterferrie** [comment]
Please share a save-state so that I can compare memory contents.

**Comment by xotmatrix** [comment]
This zip includes the save state, the disks, and a screenshot. This was practice mode but I get the same results with single event.
[Calfornia Games (4am crack) side B.aws.yaml.zip](https://github.com/a2-4am/4sports/files/10290550/Calfornia.Games.4am.crack.side.B.aws.yaml.zip)

What I noticed just now is the name is misspelled. I don't have an explanation (an earlier @a2-4am release?). I compared it to the zip on Asimov and there is a significant difference on side B, track $1D, sector $A. More digging shows it is a duplicate of sector $8, somehow written to $A by error I presume. Replacing the sector on an otherwise untouched image from the Asimov zip appears to fix the issue.

EDIT:

Oh, look at this in the crack log. I probably should have checked that first. 
```
               Changelog


2021-08-29

- re-issue to fix previously undetected
  data corruption on side B, T1D,S0A

2014-05-26

- initial release
```
I don't think that update went the way it was supposed to.

I was today years old when I realized an offset in a DSK file points to the Track/Sector like 0xTTSnn. That's what six years of paying attention gets you.
